Makefile

```diff
@@ -42,7 +42,7 @@ upgrade: $(COMMON_CONSTRAINTS_TXT)
 	pip install -qr requirements/pip-tools.txt
 	pip-compile --upgrade -o requirements/base.txt requirements/base.in
 	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in requirements/quality.in
-	pip-compile --upgrade -o requirements/doc.txt requirements/base.in requirements/doc.in
+	pip-compile --upgrade --allow-unsafe -o requirements/doc.txt requirements/base.in requirements/doc.in
```

Comment:
We should add `setuptools` as a dependency in `docs` tox env instead of allowing unsafe requirements if it works.